submissions/sayckl/friends-app/style.css

```diff
@@ -0,0 +1,96 @@
+header {
+  background-color: pink;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  border: 1px solid black;
+  border-radius: 0 0 7px 7px;
+}
+.menu-btn {
+  display: block;
+  width: 40px;
+  height: 40px;
+  background-color: #fff;
+  border-radius: 50%;
+  position: relative;
+}
+.menu-btn span,
+.menu-btn span::before,
+.menu-btn span::after {
+  position: absolute;
+  top: 50%;
+  margin-top: -1px;
+  left: 50%;
+  margin-left: -10px;
+  width: 20px;
+  height: 2px;
+  background-color: #222;
+}
+.menu-btn span::before,
+.menu-btn span::after {
+  content: '';
+  display: block;
+  transition: 0.5s;
+}
+.menu-btn span::before {
+  transform: translateY(-5px);
+}
+.menu-btn span::after {
+  transform: translateY(5px);
+}
+.menu-btn_active span {
+  height: 0;
+}
+.menu-btn_active span:before {
+  transform: rotate(45deg);
+}
+.menu-btn_active span:after {
+  transform: rotate(-45deg);
+}
+body  {
+  margin: 0;
+}
+.body {
+
+}
+aside {
```

Comment:
Styles on tags is a bad idea - they are too general. Give a class name

Reply:
aside added class